src/App.jsx

```diff
@@ -14,33 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'ALPHABET';
+const SORT_FIELD_LENGTH = 'LENGTH';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPrepearedGoods(goods, sortField, reversed) {
```

Comment:
Method's name should start with a verb. Consider renaming `getPrepearedGoods` to something like `prepareGoods`. This makes it more clear that this function is performing an action.